modules/specialized-guides/pages/large-deployments/hub-install.adoc

```diff
@@ -1,21 +1,153 @@
+
+// For a hub environment, first ... then ...
+
+
+
+[[lsd-hub-install-self-hub]]
+=== Hub Server Installation
+
+.Procedure: Installing the hub server
+
+. Install a container host with {sl-micro}.
+  For more information about {sl-micro} as a container host, see xref:installation-and-upgrade:container-deployment/suma/server-deployment-suma#deploy-suma-server-micro[].
+
+. On the container host, deploy {productname} as the hub server.
+  Add [option]``--hubxmlrpc-replicas 1`` to the [command]``mgradm install`` command line.
+  For example:
++
+
+----
+mgradm install podman suma.example.com --hubxmlrpc-replicas 1
+----
++
+
+For more information about deploying with [command]``mgradm``, see xref:installation-and-upgrade:container-deployment/suma/server-deployment-suma.adoc#deploy-suma-server-mgradm[].
+
+////
+Next step, peripheral server
+ On the container host with the hub server, prepare the SSL certificates for the peripheral servers.
+  For more information about SSL certificates, see xref:specialized-guides:large-deployments/hub-reqs.adoc#lsd-hub-reqs-certs[].
+////
+
+
+
+[[lsd-hub-install-self-peripheral]]
+=== Peripheral Servers
+
+// FIXME: make use of 'mgrctl cp'
+.Procedure: Peripheral Servers with Self-Generated Certificates
+
+. Preliminary Requirement: Hub server installation.
+  For more information, see xref:specialized-guides:large-deployments/hub-install.adoc#lsd-hub-install-self-hub[].
+. On the container host of the hub server, enter the server container with:
++
+
+----
+mgrctl term
+----
+
+
+. Inside the container, run [command]``rhn-ssl-tool`` for every pheripheral server:
++
+
+----
+rhn-ssl-tool --gen-server --dir="/root/ssl-build" --set-country="COUNTRY" \
+  --set-state="STATE" --set-city="CITY" --set-org="ORGANIZATION" \
+  --set-org-unit="ORGANIZATION UNIT" --set-email="name@example.com" \
+  --set-hostname=PERIPHAL --set-cname="example.com"
+----
+
+. For every peripheral server:
+* From the hub server container, copy [path]``/root/ssl-build/RHN-ORG-TRUSTED-SSL-CERT``,  [path]``/root/ssl-build/<hostname>/server.crt`` and [path]``/root/ssl-build/<hostname>/server.key`` to the peripheral server host.
+* On every peripheral server host, copy [path]``RHN-ORG-TRUSTED-SSL-CERT`` to [path]``/etc/pki/trust/anchors/``, and run [command]``update-ca-certificates``.
+* On every peripheral server host, deploy {productname} with:
++
+
+----
+mgradm install podman --ssl-ca-root RHN-ORG-TRUSTED-SSL-CERT --ssl-server-cert server.crt --ssl-server-key server.key
+----
+
+* Finally, on every peripheral server host, register the peripheral server to the hub server:
++
+
+// CHECKIT: did we specify the credential during the hub server deployment?
++
+----
+mgradm hub register --api-password <hub password> --api-server <hub fqdn> --api-user <hub admin>
+----
+
+
+
+== Background Information
+
+OPTIONAL: Inside the container, check that these parameters in the [path]``/etc/hub/hub.conf`` configuration file are correct:
```

Comment:
The environment variables override the `/etc/hub/hub.conf` inside the container, so looking there is misleading.

The user should look at `/etc/systemd/system/uyuni-hub-xmlrpc.service`  generated by mgradm, it looks like this:
https://github.com/uyuni-project/uyuni-tools/blob/main/mgradm/shared/templates/hubXmlrpcServiceTemplate.go#L15

and eventually customize it in `/etc/systemd/system/uyuni-hub-xmlrpc.service.d/local.conf`